### PR TITLE
SCCP-358 Arbitrum Sepolia

### PR DESCRIPTION
**Pull request by kaleb-keny**

### CODE REVIEW

File: tomls/omnibus-arbitrum-sepolia/perps/markets/imx.toml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+imx_perps_minimum_position_margin = "<%= parseEther('50') %>"
+imx_perps_locked_oi_ratio = "<%= parseEther('0.5') %>"
+imx_perps_max_liquidation_pd = "<%= parseEther('0.0005') %>"
+imx_perps_endorsed_liquidator = "0x11233749514Ab8d00C0A5873DF7428b3db70030f"
+
+[invoke.createImxPerpsMarket]
+target = ["perpsFactory.PerpsMarketProxy"]
+fromCall.func = "owner"
+func = "createMarket"
+args = ["<%= settings.imx_perps_market_id %>", "Immutable", "IMX"]
```
Comment (noisekit) [inline tomls/omnibus-arbitrum-sepolia/perps/markets/imx.toml:24]: image

```suggestion
args = ["<%= settings.imx_perps_market_id %>", "Immutable X", "IMX"]
```

File: tomls/omnibus-arbitrum-sepolia/perps/markets/ldo.toml [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+ldo_perps_minimum_position_margin = "<%= parseEther('50') %>"
+ldo_perps_locked_oi_ratio = "<%= parseEther('0.5') %>"
+ldo_perps_max_liquidation_pd = "<%= parseEther('0.0005') %>"
+ldo_perps_endorsed_liquidator = "0x11233749514Ab8d00C0A5873DF7428b3db70030f"
+
+[invoke.createLdoPerpsMarket]
+target = ["perpsFactory.PerpsMarketProxy"]
+fromCall.func = "owner"
+func = "createMarket"
+args = ["<%= settings.ldo_perps_market_id %>", "Lido dao", "LDO"]
```
Comment (noisekit) [inline tomls/omnibus-arbitrum-sepolia/perps/markets/ldo.toml:24]:
```suggestion
args = ["<%= settings.ldo_perps_market_id %>", "Lido DAO", "LDO"]
```

Actual name is `Lido DAO Token` though

File: tomls/omnibus-arbitrum-sepolia/perps/markets/meme.toml [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+meme_perps_minimum_position_margin = "<%= parseEther('50') %>"
+meme_perps_locked_oi_ratio = "<%= parseEther('0.5') %>"
+meme_perps_max_liquidation_pd = "<%= parseEther('0.0005') %>"
+meme_perps_endorsed_liquidator = "0x11233749514Ab8d00C0A5873DF7428b3db70030f"
+
+[invoke.createMemePerpsMarket]
+target = ["perpsFactory.PerpsMarketProxy"]
+fromCall.func = "owner"
+func = "createMarket"
+args = ["<%= settings.meme_perps_market_id %>", "Meme coin", "MEME"]
```
Comment (noisekit) [inline tomls/omnibus-arbitrum-sepolia/perps/markets/meme.toml:24]:
```suggestion
args = ["<%= settings.meme_perps_market_id %>", "Memecoin", "MEME"]
```

**Comment by kaleb-keny** [comment]
thank you @noisekit sir for being thorough

File: omnibus-arbitrum-sepolia.toml [hunk 4]
```diff
@@ -43,40 +43,91 @@ include = [
     # Pyth oracles
     "tomls/omnibus-arbitrum-sepolia/oracles/pyth-aave.toml",
     "tomls/omnibus-arbitrum-sepolia/oracles/pyth-ada.toml",
+    "tomls/omnibus-arbitrum-sepolia/oracles/pyth-algo.toml",
```
Comment (noisekit) [inline omnibus-arbitrum-sepolia.toml:46]: Bump the version please

**Review by noisekit — APPROVED** [review]

**Pull request merged**